### PR TITLE
Don't override databaseUrl from env var if not set

### PR DESCRIPTION
**Pull request by quvide**

### CODE REVIEW

**Review by bqqbarbhg — APPROVED** [review]
Good to go

File: server/src/setup.js [hunk 1]
```diff
@@ -101,7 +101,11 @@ function overrideConfigFromEnv() {
   if (useTestData) {
     config.useTestData = useTestData !== '0'
   }
-  config.databaseUrl = process.env.DATABASE_URL
+
+  const databaseUrl = process.env.DATABASE_URL
+  if (databaseUrl) {
+    config.databaseUrl = process.env.DATABASE_URL
```
Comment (bqqbarbhg) [inline server/src/setup.js]: Use the cached variable!

**Review by bqqbarbhg — CHANGES_REQUESTED** [review]

File: server/src/setup.js (hunk 1 above)
Comment (quvide) [reply server/src/setup.js]: good catch :D

Comment (quvide) [reply server/src/setup.js]: fixed it now

**Comment by HenrikHelenius** [comment]
@bqqbarbhg check if ok now, re-approve

**Review by bqqbarbhg — APPROVED** [review]
YES

**Comment by tobbe-j** [comment]
👍

**Pull request merged**